tests/metadata/test_metadata.py

```diff
@@ -107,21 +107,21 @@ def test_text_search_invalid_query():
         metadata_provider.text_search(text='', offset='Invalid')
 
 
-def test_query_search(asset1, asset2):
-    num_matches = 0
-    metadata_provider.publish_asset_ddo(asset1)
-
-    assert len(metadata_provider.query_search(search_query={"query": {"type": ["access"]}},
-                                              offset=10000)['results']) == (
-                   num_matches + 1)
-
-    metadata_provider.publish_asset_ddo(asset2)
-
-    assert len(metadata_provider.query_search(search_query={"query": {"type": ["access"]}},
-                                              offset=10000)['results']) == (
-                   num_matches + 2)
-    metadata_provider.retire_asset_ddo(asset1.did)
-    metadata_provider.retire_asset_ddo(asset2.did)
+# def test_query_search(asset1, asset2):
```

Comment:
Should this be removed if it's not being used?

Reply:
Sure